docs/man/docker-run.1.md

```diff
@@ -110,6 +111,14 @@ milliseconds.
 **--cidfile**=""
    Write the container ID to the file
 
+**--cpu-quota**=0
```

Comment:
Why not just set this to the default value 100? From your documentation, the interpretation is the percentage of CPU allocated or 0=100.  Then a user can just enter 50 or 25 or whatever and have your implementation actually calculate it for them.  As it is now, we force the user to understand that 0 is 100 and later 50000 is 50%....